NEWS

```diff
@@ -6,6 +6,16 @@ release. For an exhaustive list of changes, see the ``CHANGES`` file
 Zeek 4.2.0
 ==========
 
+Breaking Changes
+----------------
+
+- The existing ``Tag`` types in C++ (``zeek::Analyzer::Tag``, etc) have been merged
+  into a single type called ``zeek::Tag``. This is a breaking change, and may result
+  in plugins failing to build where they were relying on those types being different
+  for function overloading and such. We attempted to include deprecated versions of
```

Comment:
It's not just overloading or other indirect usage, right? If somebody is directly using ``zeek::Analyzer::Tag`` the code needs to be updated.

Reply:
It was more a comment on the places we were seeing problems with adding the deprecated types.

Reply:
I'll add a little extra there about moving to the new type.